docs/source/install/installation.rst

```diff
@@ -7,8 +7,8 @@ Installation
 This section of the manual will guide you through the process of installing AiiDA on your system.
 AiiDA has been tested to run on the following platforms:
 
+* Ubuntu 14.04
```

Comment:
Isn't it working anymore on 16.04?

Reply:
No it is, but below it says that it works for other Debian based distributions so I though that covered it. But I will put the explicit reference back for 16.04